src/components/GoogleAddressLookup/__test__/format.spec.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import getFormattedValue from '../helpers/getFormattedValue';
+import SelectedLocationIcon from '../icons/selectedLocationIcon';
+
+const suggestion = {
+    description: 'Cubbon Park, Sampangi Rama Nagara, Bengaluru, Karnataka, India',
+    id: 'f334fb17a44b740132f72d59a7d90ab9a81300ac',
+    matched_substrings: [{ length: 3, offset: 0 }],
+    place_id: 'ChIJL2fQ53MWrjsRuN9D6aalLMY',
+    reference: 'ChIJL2fQ53MWrjsRuN9D6aalLMY',
+    structured_formatting: {
+        main_text: 'Cubbon Park',
+        main_text_matched_substrings: [{ length: 3, offset: 0 }],
+        secondary_text: 'Sampangi Rama Nagara, Bengaluru, Karnataka, India',
+    },
+    terms: [
+        { offset: 0, value: 'Cubbon Cubbon Park' },
+        { offset: 13, value: 'Sampangi Rama Nagara' },
+        { offset: 35, value: 'Bengaluru' },
+        { offset: 46, value: 'Karnataka' },
+        { offset: 57, value: 'India' },
+    ],
+    types: ['park', 'point_of_interest', 'establishment'],
+};
+
+describe('formatSuggestionItem', () => {
+    it('should return the item formated without highligth', () => {
+        expect(getFormattedValue(suggestion, false, <SelectedLocationIcon />)).toEqual({
+            id: 'ChIJL2fQ53MWrjsRuN9D6aalLMY',
+            label: 'Cubbon Park, Sampangi Rama Nagara, Bengaluru, Karnataka, India',
+            description: 'Sampangi Rama Nagara, Bengaluru, Karnataka, India',
+            icon: <SelectedLocationIcon />,
+        });
+    });
+
+    // it('should return the item formated with hightligh', () => {
+    //     expect(getFormattedValue(suggestion, true, <SelectedLocationIcon />)).toEqual({
+    //         id: 'ChIJL2fQ53MWrjsRuN9D6aalLMY',
+    //         label: <span><b>Cub</b>bon Park</span>,
+    //         description: 'Sampangi Rama Nagara, Bengaluru, Karnataka, India',
+    //         icon: <SelectedLocationIcon />,
+    //     });
+    // });
```

Comment:
try to fix it, if not remove it and in another PR we add the tests

Reply:
this test works ... but as we tried to use jsx and avoid the use dangerousSetInnerHTML... there are several characters that React generates that cause differences between expected result and the result itself... so it is better to remove it...